kuma/wiki/models.py

```diff
@@ -1584,7 +1594,10 @@ class DocumentZone(models.Model):
     attributes inherited by the topic hierarchy beneath it.
     """
     document = models.OneToOneField(Document, related_name='zone')
-    styles = models.TextField(null=True, blank=True)
```

Comment:
We should keep styles for now, so that there are both ``styles`` and ``css_slug`` in prod push 1, and then drop ``styles`` in the next production push.  Otherwise, we'll have a flood of 500s between the migration and the code deployment.